src/Microsoft.ML.Core/Data/UInt128.cs

```diff
@@ -11,14 +11,14 @@ namespace Microsoft.ML.Runtime.Data
     /// <summary>
     /// A sixteen-byte unsigned integer.
     /// </summary>
-    public readonly struct UInt128 : IComparable<UInt128>, IEquatable<UInt128>
+    public readonly struct RowId : IComparable<RowId>, IEquatable<RowId>
     {
         // The low order bits. Corresponds to H1 in the Murmur algorithms.
         public readonly ulong Lo;
         // The high order bits. Corresponds to H2 in the Murmur algorithms.
         public readonly ulong Hi;
 
-        public UInt128(ulong lo, ulong hi)
+        public RowId(ulong lo, ulong hi)
```

Comment:
>public RowId(ulong lo, ulong hi) [](start = 8, length = 32)

Need proper summary.
I would probably renamed to lowerWord and highWord.

Reply:
If we rename the parameters of the constructor, we should rename the public fields as well. The names should be insync.